src/toxcore/state_format/old.rs

```diff
@@ -151,6 +160,37 @@ pub struct NospamKeys {
 /// Number of bytes of serialized [`NospamKeys`](./struct.NospamKeys.html).
 pub const NOSPAMKEYSBYTES: usize = NOSPAMBYTES + PUBLICKEYBYTES + SECRETKEYBYTES;
 
+/** E.g.
```

Comment:
>In the comments you should never just throw an example at the reader without a short explanation what it does or what it should be an example for :)

That's actually a doctest that compiles & runs. Aside from that, there are comments in the code sample that kinda should inform what the properties are – what do the comments miss?

![spectacle aq2064](https://cloud.githubusercontent.com/assets/3148759/19497086/72faf8ee-9582-11e6-87fa-efb1d4b1f7d4.png)